operators/multiclusterobservability/manifests/base/grafana/dash-acm-alert-analysis.yaml

```diff
@@ -633,7 +633,7 @@ data:
           "targets": [
             {
               "exemplar": true,
-              "expr": "sum(ALERTS{alertstate=\"firing\"}) by (cluster,alertname,severity)",
+              "expr": "sum(ALERTS{alertstate=\"firing\", severity=~\"$severity\"}) by (cluster, alertname, severity)",
```

Comment:
Creating a `cluster` label as you have proposed above will help in making this display better